Update introducing-metrics-in-innersource.md (#350)
gruetter/innerSourceTest: patterns/1-initial/introducing-metrics-in-innersource.md

* Another class of metrics is around how much InnerSourcing is happening. A raw definition of InnerSource based on code submissions to a repository not owned by the submitter's team could be measurable in a few ways.
* A third class of metrics are KPIs that we believe will improve the raw amount of InnerSourcing happening (e.g. mean-time-to-review, automated test runs on PRs, etc.).

> Note: The aforementioned metrics are some common examples. They aren't a one-size-fits-all. We expect the business goals of the organization to drive the metric selection process. Below is the list consolidating some metrics candidates by business goals.

Reviewer: ```suggestion
_Note:_ The aforementioned metrics are some common examples. They aren't a one-size-fits-all. We expect the business goals of the organization to drive the metric selection process. 

The list below consolidates some metrics candidates, grouped by business goals.

```

| Business Goals | Metric Candidates |
|---|---|
| Remove silos and bottlenecks | # of PRs from external contributors<br># of PRs from external contributors that are merged<br># of outside collaborators<br># of issues created from external contributors<br># of reviews from external contributors |

> | Knowledge sharing | # of clone or forks from external guests<br># of visits from external guests (proposed by Jack) |

Reviewer: Do you want to keep this "(proposed by Jack)" note in here?

We don't have to mention where which metric comes from. It is actually quite likely that many organizations have discover the same metrics over and over again anyways :)

Author: Sorry, that's typo. I should remove it from the sentence

Author: BTW, I will contact Wilson and Aaron to check if they concern on being mentioned in the page; Thanks for @spier 's review and reminding

Author: Confirmed no concern from Aaron, and waiting for Wilson's confirmation

Author: Update, just got reply from Wilson. Confirmed he has no concern on this.

| Creating reusable software | # of submodule usage<br># of code reuse across projects<br># of clone or forks from external guests |
| Increase development speed | Time to market<br>Mean-time-to-review |
| Employee satisfication& talent retention |  |
| Improve quality | # of automated test runs on PRs<br># of issues created from external contributors |
| Innovation |  |
| Others | # of repositories using InnerSource components (managed contributing guidelines, assigned TCs)<br>Days of host's responsiveness on the issues/PRs created by external contributors |

> 
> 

Reviewer: ```suggestion

```

Fix linter issue

* A third class of metrics are KPIs that we believe will improve the raw amount of InnerSourcing happening (e.g. mean-time-to-review, automated test runs on PRs, etc.).

_Note:_ The aforementioned metrics are some common examples. They aren't a one-size-fits-all. We expect the business goals of the organization to drive the metric selection process.

The list below consolidates some metrics candidates, grouped by business goals.

| Business Goals | Metric Candidates |
|---|---|
| Remove silos and bottlenecks | # of PRs from external contributors<br># of PRs from external contributors that are merged<br># of external contributors<br># of issues created from external contributors<br># of reviews from external contributors |
| Knowledge sharing | # of clone or forks from external guests<br># of visits from external guests |
| Creating reusable software | # of submodule usage<br># of code reuse across projects<br># of clone or forks from external guests |
| Increase development speed | Time to market<br>Mean-time-to-review |

> | Employee satisfication& talent retention |  |

Reviewer: ```suggestion
| Employee satisfaction & talent retention |  |
```

Fix typo and whitespace

| Business Goals | Metric Candidates |
|---|---|
| Remove silos and bottlenecks | # of PRs from external contributors<br># of PRs from external contributors that are merged<br># of outside collaborators<br># of issues created from external contributors<br># of reviews from external contributors |
| Knowledge sharing | # of clone or forks from external guests<br># of visits from external guests (proposed by Jack) |
| Creating reusable software | # of submodule usage<br># of code reuse across projects<br># of clone or forks from external guests |
| Increase development speed | Time to market<br>Mean-time-to-review |
| Employee satisfication& talent retention |  |
| Improve quality | # of automated test runs on PRs<br># of issues created from external contributors |
| Innovation |  |
| Others | # of repositories using InnerSource components (managed contributing guidelines, assigned TCs)<br>Days of host's responsiveness on the issues/PRs created by external contributors |

> ## Resulting Context

Reviewer: ```suggestion

## Resulting Context
```

- Clint
- Russ Rutledge
- Tom
- Jack Yang

## Acknowledgement

- Georg
- Bob
- Aaron Steward

> - Willson Mar

Reviewer: ```suggestion
- Wilson Mar
```

Possibly spelled like this?

Also did you check with both Aaron and Wilson that they would like to be mentioned here?

- Clint
- Russ Rutledge
- Tom
- Jack Yang

## Acknowledgement

- Georg
- Bob
- [Aaron Stewart](https://github.com/a-a-ron/innersource-template-pluralsight/tree/master/metrics)

> - Wilson Mar

Reviewer: @arlou do you have any online resource by Wilson Mar that you want to reference here?

Author: hi @spier, thanks for checking; So far, no referred link on Wilson as I referred to his inputs from Slack channel.
